src/sagas/featureToggle.js

```diff
@@ -123,6 +125,7 @@ export function* monitorFeatureFlags(currentRetry = 0) {
     const featureToggles = mapFeatureToggles(unleashClient.getToggles());
 
     yield put(setFeatureToggles(featureToggles));
+    yield put(setUseSafeBiometryMode(featureToggles[SAFE_BIOMETRY_MODE_FEATURE_TOGGLE]));
```

Comment:
There are 3 state keys that are created from feature flags that require a special handling, the ones controlling wallet-service, push-notification and safe-biometry-mode.

The first 2 can safely be recreated during the startWallet saga ([reference](https://github.com/HathorNetwork/hathor-wallet-mobile/blob/feat/improved-pin-security-migration/src/sagas/wallet.js#L194-L195)) after the reset, but since the safe biometry mode is required to render the first screen after the reset we need to port its state in the reset.